lib/queries.js

```diff
@@ -140,11 +158,17 @@ export async function runQuery(query) {
 export async function rawQuery(query, params = []) {
   const db = getDatabase();
 
-  if (db !== POSTGRESQL && db !== MYSQL) {
+  if (db !== POSTGRESQL && db !== MYSQL && db !== SQLITE) {
     return Promise.reject(new Error('Unknown database.'));
   }
 
-  const sql = db === MYSQL ? query.replace(/\$[0-9]+/g, '?') : query;
+  let sql = db !== POSTGRESQL ? query.replace(/\$[0-9]+/g, '?') : query;
+  
+  if (db === SQLITE) {
+    params[1] = params[1].getTime();
+    if (params.length > 2) params[2] = params[2].getTime();
+    if (sql.match(/concat\(/)) sql = sql.replace(/concat\(.*?\)/, "event_type || '\t' || event_value");
```

Comment:
I think a completely separate raw query for sqlite would be better than doing this replacement. It will be hard to maintain in the future. Otherwise looks great!

Reply:
Thanks! Multiple ways of doing it separatly, I'd rather let you handle it the way you want.
Especially as you just refactored the queries structure.